input/vulnerabilities/Defy_republic_init_runit.json

```diff
@@ -31,5 +31,5 @@
   "Target": ["system-component"],
   "Channel": ["app-execution"],
   "Condition": ["affected-app-installed"],
-  "Privilege": ["root"]
```

Comment:
This vulnerability sounds like it should be "root" as it exploits init which is a userspace root daemon. I could be wrong.

Reply:
That looks right having had another look - now changed